submitted_models/coro_karen_sensor_config_1/launch/description.launch

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="name" doc="Name of Vehicle"/>
+  <param name="$(arg name)/robot_description" command="$(find xacro)/xacro '$(find coro_karen_sensor_config_1)/urdf/karen_from_sdf.xacro' name:=$(arg name)"/>
```

Comment:
I'm not able to visualize `RobotModel` type visualization in RViz.  this is the case on Allie as well while being related [the thread on Allie](https://github.com/osrf/subt/pull/813#discussion_r601572455).

Reply:
Since the xacro / urdf is optional I went ahead and disabled it in the description.launch.  I will add it to our task list as an improvement that can be done later.